src/rculfhash.c

```diff
@@ -568,6 +568,43 @@ unsigned int cds_lfht_fls_ulong(unsigned long x)
 #endif
 }
 
+static
+void *cds_lfht_malloc(void *state __attribute__((unused)), size_t size)
+{
+	return malloc(size);
+}
+
+static
+void *cds_lfht_calloc(void *state __attribute__((unused)), size_t nmemb, size_t size)
+{
+	return calloc(nmemb, size);
+}
+
+static
+void *cds_lfht_realloc(void *state __attribute__((unused)), void *ptr, size_t size)
+{
+	return realloc(ptr, size);
+}
+
+static
+void *cds_lfht_aligned_alloc(void *state __attribute__((unused)), size_t alignment, size_t size)
+{
+	return aligned_alloc(alignment, size);
+}
+
+static
+void cds_lfht_free(void *state __attribute__((unused)), void *ptr)
+{
+	free(ptr);
+}
+
+
+/* Default memory allocator */
+static
+struct cds_lfht_alloc cds_lfht_default_alloc =
+		{.malloc = cds_lfht_malloc, .calloc = cds_lfht_calloc, .realloc = cds_lfht_realloc,
+		 .aligned_alloc = cds_lfht_aligned_alloc, .free = cds_lfht_free, .state = NULL};
+
```

Comment:
style:

static struct blah name = {
	.field = val,
	.field2 = val2,
};

Reply:
Just to make sure I don't have to change this several times, what is the recommended style for multi-line? Would something like the following work?

```
static struct blah name = { 
    .field = val, 
    .field2 = val2, 
};
```

Reply:
Yes, that's good.

Reply:
Fixed